ee/localserver/request-id.go

```diff
@@ -68,6 +72,9 @@ func (ls *localServer) requestIdHandler() http.Handler {
 }
 
 func (ls *localServer) requestIdHandlerFunc(res http.ResponseWriter, req *http.Request) {
+	_, span := traces.New(req.Context(), trace.WithAttributes(attribute.String(traces.AttributeName("localserver", "path"), req.URL.Path)))
```

Comment:
i wonder if the ctx should replace the req one

Reply:
I don't think it matters

Reply:
This is the innermost handler -- there shouldn't be any more child spans

Reply:
For now. :)

(probably fine. but I want to kinda note it as almost an antipattern)